Components.Tests/RenderTests/RenderSplashInfoTests.cs

```diff
@@ -15,9 +15,11 @@ public void RenderSplashInfoTest(Model sut)
             sut.SplashPageInfo.FrenchTermsUrl = "http://www.canada.ca/fr/transparence/avis.html";
             sut.SplashPageInfo.EnglishName = "[My web asset]";
             sut.SplashPageInfo.FrenchName = "[Mon actif web]";
+            sut.Settings.LeavingSecureSiteWarning.Enabled = false;
```

Comment:
Not necessarily something wrong, but just so I am sure:  LeavingSecureSiteWarning.Enabled's default is `false` and SRIEnabled's default is `true`.  In the case of LeavingSecureSiteWarning NSubsitute assigns true and in the case of SRIEnabled it assigns false?   Does it just alternate between true and false?   (I guess my question is, why do we need to specifically set both?)

Reply:
Will investigate after